### PR TITLE
[TE] Fix bug in AutoInlineElemWise and implement AutoInlineBroadcast

### PR DESCRIPTION
**Pull request by leeexyz**
Hi Community,

When I try to use the auto inline functions, I found that one for element-wise is not working due to the wrong **VisitExpr_**, it should be **ProducerLoadNode** not **CallNode**. And I also implement the broadcast one by simply checking the **tag**, cause ops in Topi has the specified tag.

@comaniac @ZihengJiang @tqchen Please help to review, thanks!

### CODE REVIEW

File: src/te/schedule/auto_inline_elem_wise.cc [hunk 1]
```diff
@@ -113,6 +117,8 @@ void AutoInlineInjective(Schedule sch) {
 
 TVM_REGISTER_GLOBAL("schedule.AutoInlineElemWise").set_body_typed(AutoInlineElemWise);
 
+TVM_REGISTER_GLOBAL("schedule.AutoInlineBroadcast").set_body_typed(AutoInlineBroadcast);
```
Comment (jcf94) [inline src/te/schedule/auto_inline_elem_wise.cc:120]: Also add an interface to `tvm/te/schedule_pass.h`.

**Review by jcf94 — COMMENTED** [review]
Looks good.

File: src/te/schedule/auto_inline_elem_wise.cc (hunk 1 above)
Comment (leeexyz) [reply src/te/schedule/auto_inline_elem_wise.cc:120]: > Also add an interface to `tvm/te/schedule_pass.h`.

Thanks! The API was added.

File: tests/python/unittest/test_te_schedule_ops.py [hunk 2]
```diff
@@ -110,19 +110,53 @@ def argmax_init(idx_typ, val_typ):
...
+    def check_auto_inline(schedule_func, auto_inline_func):
+        s, T1 = schedule_func()
+        # before auto inline the attach type is AttachType.kGroupRoot
+        assert s[T1].attach_type == 1
+        auto_inline_func(s)
+        # after auto inline the attach type is AttachType.kInline
+        assert s[T1].attach_type == 2
+        s = s.normalize()
+        bounds = tvm.te.schedule.InferBound(s)
+        stmt = tvm.te.schedule.ScheduleOps(s, bounds)
```
Comment (comaniac) [inline tests/python/unittest/test_te_schedule_ops.py:155]: Is there any way to check `stmt`?

Comment (leeexyz) [reply tests/python/unittest/test_te_schedule_ops.py:155]: > Is there any way to check `stmt`?

Yes, check **IR** structure is an option, like if the inlined ProducerRealizeNode is still there. But is it a bit redundant? Cause we already checked the **attach_type**.

**Review by jcf94 — APPROVED** [review]

**Review by comaniac — APPROVED** [review]

**Comment by comaniac** [comment]
Thanks @leeexyz @jcf94

**Pull request merged**